src/Hal/Serializer/ItemNormalizer.php

```diff
@@ -81,6 +81,8 @@ public function supportsDenormalization($data, $type, $format = null)
      * {@inheritdoc}
      *
      * @throws RuntimeException
+     *
+     * @return object
```

Comment:
This is the only weird thing in this PR. Lol...

Because PHP-CS-Fixer only knows about what's in the current file. (It does not parse parent classes / interfaces.)

And of course when we bump to PHP 7.2, we can add the `object` return type.